scripts/flu_hosp_prod.R

```diff
@@ -264,7 +264,25 @@ rlang::list2(
           select(geo_value, source, target_end_date = time_value, value) %>%
           filter(target_end_date > truth_data_date, geo_value %nin% insufficient_data_geos) %>%
           mutate(target_end_date = target_end_date + 6)
-        truth_data <- nhsn_latest_data %>%
+        forecast_date <- as.Date(forecast_generation_date)
+        if (forecast_date < Sys.Date()) {
+          truth_data <- nhsn_archive_data %>% epix_as_of(forecast_date)
+        } else {
+          truth_data <- nhsn_latest_data
+        }
+        nhsn_latest_data %>%
+          pull(time_value) %>%
+          max()
+        nhsn_archive_data %>%
+          epix_as_of(as.Date("2024-12-18")) %>%
+          pull(time_value) %>%
+          max()
+        forecast_date
+        nhsn_archive_data %>%
+          epix_as_of(forecast_date) %>%
+          pull(time_value) %>%
+          max()
```

Comment:
Debug stuff

Reply:
good catch. that target is kind of a mess at this point, and should probably be split into functions